12.8-2.3/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM postgres:12.8
+
+ENV PLV8_VERSION=2.3.15 \
+    PLV8_SHASUM="8a05f9d609bb79e47b91ebc03ea63b3f7826fa421a0ee8221ee21581d68cb5ba"
+
+RUN buildDependencies="build-essential \
+    ca-certificates \
+    curl \
+    git-core \
+    python \
+    python3 \
+    gpp \
+    cpp \
+    pkg-config \
+    apt-transport-https \
+    cmake \
+    libc++-dev \
+    libc++abi-dev \
+    postgresql-server-dev-$PG_MAJOR \
+    postgresql-contrib" \
```

Comment:
I am getting an error during the image building process when the `postgresql-contrib` dependency is included, is this mandatory?

Reply:
I believe so because it contains headers PLV8 needs to compile itself. I don't remember if you need the lib package as well?

I forgot to mention earlier, you'll want to use PostgreSQL APT repo instead of Ubuntu's. That one should "just work": https://www.postgresql.org/download/linux/ubuntu/

Reply:
@churtado-caylent What error is it?

Reply:
I didnt get an error with `postgresql-contrib`, only when it got to compiling the plv8 step. 

Also based on the link @misham posted, it looks like we would only need to install `apt-get install postgresql-12` and we would get a bunch of stuff that we are installing individually:

- postgresql-client-12
- postgresql-12
- libpq-dev
- postgresql-server-dev-12